variables.tf

```diff
@@ -29,7 +29,7 @@ variable "description" {
 
 variable "region" {
   description = "update"
-  default = "US"
+  default     = "US"
```

Comment:
Why the extra space? Original line looked fine?

Reply:
`terraform fmt` formats the script this way